benchmarks/benchmark_event_db.cpp

```diff
@@ -0,0 +1,104 @@
+#include <viewer/event_db.hpp>
+#include <viewer/base_ui.hpp>
+#include <hawktracer.h>
+#include <benchmark/benchmark.h>
+#include <hawktracer/parser/file_stream.hpp>
+#include <hawktracer/parser/protocol_reader.hpp>
+#include <hawktracer/parser/make_unique.hpp>
+
+using WellKnownKlasses = HawkTracer::parser::WellKnownKlasses;
+
+static void fnc_bar(HT_Timeline* timeline)
+{
+    HT_TP_SCOPED_STRING(timeline, "bar");
+}
+
+static void fnc_foo(HT_Timeline* timeline)
+{
+    HT_TP_SCOPED_STRING(timeline, "foo");
+    for (int i = 0; i < 10; i++)
+    {
+        fnc_bar(timeline);
+    }
+}
+
+static void fnc_start(HT_Timeline* timeline)
```

Comment:
can we have more descriptive name here? `generate_test_tracepoints` or something like this